test/common/http/conn_manager_impl_fuzz_test.cc

```diff
@@ -76,7 +76,15 @@ class FuzzConfig : public ConnectionManagerConfig {
   }
 
   void newStream() {
-    codec_ = new NiceMock<MockServerConnection>();
+    if (!codec_) {
+      codec_ = new NiceMock<MockServerConnection>();
+    } else {
+      // Use the same address for codec_. Otherwise, this codec_ is not used to dispatch from the
```

Comment:
Why can't we keep the same codec for the lifetime of the connection?

Reply:
Yeah, I could only instantiate the codec once and re-use, i guess it makes sense since it's just a mock object. That works too!